src/main/resources/application.properties

```diff
@@ -1 +1 @@
-server.port=0
+server.port=8080
```

Comment:
why change server port to a fixed value?

Reply:
done